contrib/test.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+set -ex
+
+FEATURES="serde rand all-languages"
+
+cargo --version
+rustc --version
+
+# Work out if we are using a nightly toolchain.
+NIGHTLY=false
+if cargo --version | grep nightly; then
+    NIGHTLY=true
+fi
```

Comment:
We don't need this, in other rust-bitcoin scripts this is used for error reporting but its a bit clunky and doesn't add that much value IMO.

Found with `shellcheck contrib/test.sh`

Reply:
will fix in followup